### PR TITLE
refactor: refact some code and clear some Scala code format alarm

### PR DESCRIPTION
**Pull request by jackxu2011**
### What is the purpose of the change

refact some code and clear some Scala code format alarm
### Related issues/PRs

Related issues: #2703

### Checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [Linkis mailing list](https://linkis.apache.org/community/how-to-subscribe) first)
- [ ] **If this is a code change**: I have written unit tests to fully verify the new behavior.

### CODE REVIEW

**Review by jacktao007 — APPROVED** [review]
LGTM.

File: linkis-commons/linkis-common/src/main/scala/org/apache/linkis/common/ServiceInstance.scala [hunk 1]
```diff
@@ -25,12 +25,9 @@ class ServiceInstance {
...
   def getInstance: String = instance
 
-  def canEqual(other: Any): Boolean = other.isInstanceOf[ServiceInstance]
-
   override def equals(other: Any): Boolean = other match {
     case that: ServiceInstance =>
-      (that canEqual this) &&
-        applicationName == that.applicationName &&
+      applicationName == that.applicationName &&
         instance == that.instance
```
Comment (casionone) [inline linkis-commons/linkis-common/src/main/scala/org/apache/linkis/common/ServiceInstance.scala:31]: @jacktao007 @peacewong help  check  this

File: linkis-computation-governance/linkis-manager/linkis-label-common/src/main/java/org/apache/linkis/manager/label/utils/LabelUtils.java [hunk 2]
```diff
@@ -54,24 +64,12 @@ public class LabelUtils {
...
-  private static boolean isWrapClass(Class<?> clz) {
-    try {
-      return ((Class<?>) clz.getField("TYPE").get(null)).isPrimitive();
-    } catch (Exception e) {
+    if (clz == null) {
       return false;
     }
+    return clz.equals(String.class)
+        || clz.equals(Enum.class)
+        || org.apache.commons.lang3.ClassUtils.isPrimitiveOrWrapper(clz);
```
Comment (casionone) [inline linkis-computation-governance/linkis-manager/linkis-label-common/src/main/java/org/apache/linkis/manager/label/utils/LabelUtils.java:72]: lgtm

**Comment by peacewong** [comment]
It's recommended that this PR only do this:[refact some code and clear some Scala code format alarm]

**Comment by jackxu2011** [comment]
> It's recommended that this PR only do this:[refact some code and clear some Scala code format alarm]

ok

**Review by peacewong — APPROVED** [review]
LGTM.

**Pull request merged**